resource/labeling-app/components/category-selector.js

```diff
@@ -214,8 +214,8 @@ let CategorySelector = {
                         <div class="annotation-thumb-container">
                             <AnnotationThumb :data="a" :color="labelColor(a.label)"></AnnotationThumb>
                         </div>
-                        <div v-if="!a.label">Assign a category</div>
-                        <div v-if="a.label">{{categories[a.label].caption}}</div>
+                        <div v-if="a.label && categories[a.label]">{{categories[a.label].caption}}</div>
```

Comment:
same comment about `a.label in categories` here

Reply:
Used a variable that will replace empty key. Fixed in e72c6f0